src/main/kotlin/no/nav/syfo/dialogmote/api/domain/DialogmotedeltakerArbeidsgiverDTO.kt

```diff
@@ -3,8 +3,8 @@ package no.nav.syfo.dialogmote.api.domain
 data class DialogmotedeltakerArbeidsgiverDTO(
     val uuid: String,
     val virksomhetsnummer: String,
-    val lederNavn: String?,
-    val lederEpost: String?,
+//    val lederNavn: String?,
```

Comment:
Var det sånn at disse to feltene skulle fjernes eller vil de populeres med navn og epost på nåværende nærmeste leder?
Jeg tror vi håndterer at de mangler i syfomodiaperson nå, men hvis de skal fjernes bør vi fikse https://trello.com/c/S4jjSjCq/494-se-over-og-evt-endre-bruk-av-ledernavn-fra-isidalogmote-i-syfomodiaperson før MVP går live.

Reply:
lederEpost kan i hvert fall fjernes både her og andre steder i koden.

lederNavn burde enten fjernes her og så referanser til det i syfomodiaperson. Eller så kan den evt. mappes inn her når man lager DTO-objektet.(heller enn å mappe legger ned i laget om det ikke trengs der).